sensor_control/navigator_lidar_oa/src/lidar.cpp

```diff
@@ -66,24 +65,11 @@ uf_common::PoseTwistStamped waypoint_enu,carrot_enu;
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-float LLA_BOUNDARY_X1 = -30, LLA_BOUNDARY_Y1 = 50;
-float LLA_BOUNDARY_X2 = -30, LLA_BOUNDARY_Y2 = -20;
-float LLA_BOUNDARY_X3 = 35, LLA_BOUNDARY_Y3 = -20;
-float LLA_BOUNDARY_X4 = 35, LLA_BOUNDARY_Y4 = 50;
-
-
-/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-
-/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-typedef actionlib::SimpleActionServer<uf_common::MoveToAction> MOVE_TO_SERVER;
-MOVE_TO_SERVER *actionServerPtr;
-
-void actionExecute(const uf_common::MoveToGoalConstPtr& goal)
-{
-	//Grab new goal from actionserver
-	ROS_INFO("LIDAR: Following new goal from action server!");
-	waypoint_enu.posetwist = goal->posetwist; 
-}
+//Do these come from a rosparam or service call?
+Eigen::Vector2d BOUNDARY_CORNER_1 (30, 10);
+Eigen::Vector2d BOUNDARY_CORNER_2 (30, 120);
+Eigen::Vector2d BOUNDARY_CORNER_3 (140, 120);
+Eigen::Vector2d BOUNDARY_CORNER_4 (140, 10);
 
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
I didn't have this implement yet, but it would be nice if this could get the data directly from the rosservice: /get_bounds